ibc_data/ibc_conditions.tsv

```diff
@@ -0,0 +1,380 @@
+WardAndAllport	planning_unambiguous_intermediate	Unambiguous goal state with intermediate movement
+BiologicalMotion1	global_upright	"Structural information is preserved, but individual local trajectories are mirror-symmetric. ""Global-only"" in the original paper."
+BiologicalMotion1	global_inverted	Global walker, inverted upside-down
+BiologicalMotion1	natural_upright	"Local information is preserved, but the points are randomly shuffled along the X-axis, rendering global cues uninformative. ""Local-natural"" in the original experiment."
+BiologicalMotion1	natural_inverted	Local natural walker, inverted along the horizontal axis
+BiologicalMotion2	modified_upright	"Neither structural or local information is carried out by this type of walker, it uses both types of modifications used for the previous two categories. ""Local modified"" in the original study."
+BiologicalMotion2	modified_inverted	Local modified walker, inverted along the horizontal axis
+BiologicalMotion2	natural_upright	"Local information is preserved, but the points are randomly shuffled along the X-axis, rendering global cues uninformative. ""Local-natural"" in the original experiment."
+BiologicalMotion2	natural_inverted	Local natural walker, inverted along the horizontal axis
+MathLanguage	arithmetic_fact_auditory	Listen to arithmetic fact
+MathLanguage	arithmetic_fact_visual	Read arithmetic fact
+MathLanguage	arithmetic_principle_auditory	Listen to arithmetic principle
+MathLanguage	arithmetic_principle_visual	Read arithmetic principle
+MathLanguage	colorlessg_auditory	Jabberwocky sentence presented as auditory stimulus
+MathLanguage	colorlessg_visual	Jabberwocky sentence presented as visual stimulus
+MathLanguage	context_auditory	Fixation period
+MathLanguage	context_visual	Fixation period
+MathLanguage	general_auditory	Listen to sentence
+MathLanguage	general_visual	Read sentence
+MathLanguage	geometry_fact_auditory	Listen to geometric fact
+MathLanguage	geometry_fact_visual	Read geometric fact
+MathLanguage	theory_of_mind_auditory	Listen to false-belief tale
+MathLanguage	theory_of_mind_visual	Read false-belief tale
+MathLanguage	wordlist_auditory	Listen to word list
+MathLanguage	wordlist_visual	Read word list
+SpatialNavigation	encoding_phase	Encode location of key building
+SpatialNavigation	intersection	Participant at an intersection in the city.
+SpatialNavigation	retrieval	Retrieve location of key building
+SpatialNavigation	control	Camera approaches intersection in an control trial
+SpatialNavigation	pointing_control	Participant rotates camera to point to blue building in control trial
+SpatialNavigation	experimental	Camera approaches intersection in an experimental trial.
+SpatialNavigation	pointing_experimental	Participant rotates camera to point to key building in experimental trial
+SpatialNavigation	navigation	Camera approaches intersection during training
+EmoMem	neutral_image	Neutral background image
```

Comment:
```suggestion
EmoMem	neutral_image	Neutral image
``` (I don't see why it should be a "background" image ? see also l. 252 and 253)

Reply:
I think it's because the emotional manipulation for this task only affects the background image, and not the image that is superimposed later.

Reply:
Can you make it sure (eg by replaying the stims) ? I'd like to have these annotation as clear and accurate as possible .